Update index.md (#90)
igorkamyshev/withease: apps/website/docs/factories/index.md

### `invoke`

Anywhere in your application you can invoke a factory by calling `invoke` with a factory and its arguments:

> We do not call `invoke` in components; instead, we call it in **`.js`** or **`.ts`** files, as if we were writing regular Effector code.

Reviewer: ```suggestion
It is recommended not to call ⁠invoke within components; instead, it should be called in ⁠`.js` or ⁠`.ts` files, as if writing standard Effector code.
```

> It is recommended not to call ⁠invoke within components; instead, it should be called in ⁠`.js` or ⁠`.ts` files, as if writing standard Effector code.

Reviewer 2: 1. Предлагаю выделить предупреждение через `::: warning`
2. Предлагаю удалить упоминание `.js` или `.ts` файлов — это очень фреймворк специфик информация. Думаю, в примере ниже специфичном для Реакта это указать (`Now we can use`).

Author: добавил

```ts
import { invoke } from '@withease/factories';

const { $counter, increment, decrement } = invoke(createCounter, { initialValue: 2 });
```

> Now we can use `$counter`, `increment`, and `decrement` in our components

Reviewer 2: Давай добавим сюда коллапс-секцию с заголовком «например с Реактом это использовать можно так» или что-то такое. Хочется получить текст, который не будет акцентировать внимание на конкретном фреймворке.

Author: добавил дополнительно vue, solid

Here’s how you might use them in a React component:

```jsx
const CounterComponent = () => {

>   const counter = useStore($counter);

Reviewer: ```suggestion
  const counter = useUnit($counter);
  const [onIncrement, onDecrement] = useUnit([increment, decrement])
```

Author: > Fixed, thanks!

  return (
    <div>
      <p>Counter: {counter}</p>

>       <button onClick={() => increment()}>Increment</button>
>       <button onClick={() => decrement()}>Decrement</button>

Reviewer: ```suggestion
      <button onClick={() => onIncrement()}>Increment</button>
      <button onClick={() => onDecrement()}>Decrement</button>
```

:::

```ts
import { invoke } from '@withease/factories';

const { $counter, increment, decrement } = invoke(createCounter, { initialValue: 2 });
```

Now we can use `$counter`, `increment`, and `decrement` in our components. Here’s how you might use them in different UI frameworks:

::: details Example usage in React
```jsx
const CounterComponent = () => {
  const counter = useUnit($counter);
  const [onIncrement, onDecrement] = useUnit(increment, decrement);
  
  return (
    <div>
      <p>Counter: {counter}</p>
      <button onClick={() => onIncrement()}>Increment</button>
      <button onClick={() => onDecrement()}>Decrement</button>
    </div>
  );
};
```
:::

::: details Example usage in Vue
```html
<template>
  <div>
    <p>Counter: {{ counter }}</p>
    <button @click="increment">Increment</button>
    <button @click="decrement">Decrement</button>
  </div>
</template>

<script setup>
import { useUnit } from 'effector-vue';
import { $counter, increment, decrement } from './store'; // assuming you've invoked your factory in `store.js`

const counter = useUnit($counter);
</script>
```
:::

::: details Example usage in Svelte
```svelte
<script>

>   import { useUnit } from 'effector-svelte';

Reviewer 2: Но ведь пакета `effector-svelte` не существует?..

Author: удалил, да, по svelte ошибся
